creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -282,6 +282,8 @@ In `bridge` mode, virtual machines are connected to the network backend
 through a linux ``bridge''. The pod network IPv4 address is delegated to
 the virtual machine via DHCPv4. The virtual machine should be configured
 to use DHCP to acquire IPv4 addresses.
+MAC address is also delegated if a specific MAC is not configured in the vm spec
+using DHCPv4 option 26 and also via the virtual machine domxml.
```

Comment:
That reads as if we pass through the mac via `26`. Did you write that we also pass through the MTU via dhcp?

Reply:
I share @rmohr puzzlement regarding the English text. It is not clear to me.

@SchSeba can you try to rephrase? Can you also set topic/network label here?